snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/datastore/request/rf2/exporter/Rf2RefSetExporter.java

```diff
@@ -257,7 +257,15 @@ protected SnomedRefSetMemberSearchRequestBuilder createSearchRequestBuilder() {
 					// Append extra columns using the properties map
 					final Map<String, Object> properties = member.getProperties();
 					for (final String extraColumn : extraColumns) {
-						builder.add(toColumn(properties.get(extraColumn)));
+						Object property = properties.get(extraColumn);
+						if (extraColumn.equals(SnomedRf2Headers.FIELD_QUERY)) {
+							String query = (String) property;
+							query = query.replace("\r", " ");
```

Comment:
The first replacement could be `query = query.replace("\r\n", " ");`
The way it currently is, CRLF sequences get replaced by two spaces, one for `\r` and one for `\n`.

Reply:
Another option is to use Guava's  `CharMatcher` for all replacements:
```
property = CharMatcher.breakingWhitespace().replaceFrom(query, ' ');
```